From bad81ba6684c2879e2ec595ae2a412b3b02602fc Mon Sep 17 00:00:00 2001
From: wjh <wjh@rock-chips.com>
Date: Tue, 27 Sep 2016 17:33:53 +0800
Subject: [PATCH] drivers: sound: usb: fix disvr usb Audio bug

The disvr usb audio sampling rate is through nanoc reported to
the kernel, so don't need the kernel again set the sampling rate.

Change-Id: I60409fc579952a196c4fe40f678e87d505a7508d
Signed-off-by: wjh <wjh@rock-chips.com>
---
 sound/usb/stream.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sound/usb/stream.c b/sound/usb/stream.c
index 3b23102230c0..656276398516 100644
--- a/sound/usb/stream.c
+++ b/sound/usb/stream.c
@@ -685,6 +685,9 @@ int snd_usb_parse_audio_interface(struct snd_usb_audio *chip, int iface_no)
 
 		switch (chip->usb_id) {
 		case USB_ID(0x0a92, 0x0053): /* AudioTrak Optoplay */
+#ifdef CONFIG_HID_RKVR
+		case USB_ID(0x071B, 0x3205): /* RockChip NanoC VR */
+#endif
 			/* Optoplay sets the sample rate attribute although
 			 * it seems not supporting it in fact.
 			 */
-- 
2.35.3

